### DISCUSSION TITLE
部署到CloudBase，访问应用提示404和400

### DISCUSSION THREAD

**Discussion by PoplarTang** [walinejs/waline#1179] [Q&A] [answered]
serverURL1：https://waline-poplar-0gobt3kh922c927a-1251502897.ap-shanghai.app.tcloudbase.com/waline
serverURL2：https://waline-poplar-0gobt3kh922c927a-1251502897.tcloudbaseapp.com

尝试过这两个页面提供的部署到云开发按钮都一样的结果：
https://waline.js.org/guide/server/cloudbase.html
https://github.com/walinejs/tcb-starter

**如下：**
![image](https://user-images.githubusercontent.com/4375208/177907949-5a22e5d5-dfb2-42fd-bc70-9ea11d849ad2.png)

400 BadRequest
![image](https://user-images.githubusercontent.com/4375208/177907759-c3fde85b-1291-4b4f-8c87-e9825e064295.png)

404 Not Found
![image](https://user-images.githubusercontent.com/4375208/177908334-be2937bd-0562-4362-9de9-8ab8cde9a519.png)

**Comment by lizheming** [comment, marked as answer]
same issue https://github.com/walinejs/waline/issues/1174
